### PR TITLE
KZOO-309: Add PIDF+LO XML payload builders

### PR DESCRIPTION
**Pull request by harenson**
Related to https://github.com/2600hz/kazoo-stepswitch/pull/42

When building the XML for FreeSWITCH, if a _Pidflo-Xml_ CCV exist within the resource, add a new `sip_multipart_pidflo` variable with the _Pidflo-Xml_ CCV's content.

### CODE REVIEW

**Comment by harenson** [comment]
Moved to https://github.com/2600hz/kazoo-stepswitch/pull/42

File: src/ecallmgr_fs_xml.erl [hunk 1]
```diff
@@ -865,6 +865,9 @@ kazoo_var_to_fs_var_fold(<<"Media-Control">>, Variables, Acc) ->
 kazoo_var_to_fs_var_fold(<<"Channel-Actions">>, Actions, Acc) ->
     [Actions |Acc];
 
+kazoo_var_to_fs_var_fold(<<"Pidflo-Xml">>, PIDFLO_XML, Vars) ->
+    [encode_fs_val("sip_multipart_pidflo",  PIDFLO_XML) | Vars];
```
Comment (sanjeevie6353) [inline src/ecallmgr_fs_xml.erl]: Just wondering whether the naming convention for the variable `PIDFLO_XML` can be changed to something like `PidfloXML`

Comment (harenson) [reply src/ecallmgr_fs_xml.erl]: @sanjeevie6353 Both, PIDFLO and XML, are acronyms, that is why both are in uppercase and underscore-separated.

Comment (sanjeevie6353) [reply src/ecallmgr_fs_xml.erl]: Got it.. I'll set this as resolved...

**Review by lazedo — APPROVED** [review]

**Pull request merged**